akka-docs/src/main/paradox/typed/actor-discovery.md

```diff
@@ -1,6 +1,6 @@
 # Actor discovery
 
-For the Akka Classic documentation of this feature see @ref:[Classic Actors](../actors.md#actorselection).
+The current recommended API is fully typed, a.k.a. `akka-typed`.For the Akka Classic documentation of this feature see @ref:[Classic Actors](../actors.md#actorselection).
 
```

Comment:
I think this is better for information passing

Reply:
I'm not sure I agree this improves on things. Also, each classic pages has a big box up top saying to use the new APIs, I can't see how anyone can read the docs and miss that, see https://doc.akka.io/docs/akka/current/actors.html for example.

Reply:
Yes and no,but I think we can move on without it.